neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -97,7 +97,17 @@ public virtual Fixed8 NetworkFee
 
         public virtual Fixed8 SystemFee => Settings.Default.SystemFee.TryGetValue(Type, out Fixed8 fee) ? fee : Fixed8.Zero;
 
-        public Fixed8 FeePerByte => NetworkFee / Size;
+        private Fixed8 _feePerByte = null;
+        
+        public Fixed8 FeePerByte
+        {
+            get
+            {
+                if(_feePerByte == null)
+                    _feePerByte = NetworkFee / Size;
+                return _feePerByte;
+            }
+        }
```

Comment:
I've seen things like this in many classes... it's nice, a typical C# thing, never seen on C++ before. Guess it should work :)